### PR TITLE
CTOR-990 : Refresh network-switchs-hirschmann-snmp

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh network-switchs-hirschmann-snmp

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md [hunk 1]
```diff
@@ -0,0 +1,569 @@
+---
+id: network-switchs-hirschmann
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md:2]:
```suggestion
id: network-switchs-hirschmann-snmp
```

File: pp/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md [hunk 2]
```diff
@@ -0,0 +1,568 @@
+---
+id: network-switchs-hirschmann
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md:2]:
```suggestion
id: network-switchs-hirschmann-snmp
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md [hunk 3]
```diff
@@ -0,0 +1,569 @@
+---
+id: network-switchs-hirschmann-snmp
+title: Hirschmann switch SNMP
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Dépendances du Connecteur de supervision
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md:8]:
```suggestion
## Dépendances du connecteur de supervision
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md [hunk 4]
```diff
@@ -0,0 +1,569 @@
...
+
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **Net-Hirschmann-Standard-SNMP-custom** est utilisé.
+
+</TabItem>
+<TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">
+
+| Alias                | Modèle de service                                        | Description                                       |
+|:---------------------|:---------------------------------------------------------|:--------------------------------------------------|
+| Configuration        | Net-Hirschmann-Standard-Configuration-SNMP-custom        | Contrôle l'état de la configuration               |
+| Traffic-Generic-Id   | Net-Hirschmann-Standard-Traffic-Generic-Id-SNMP-custom   | Contrôle le traffic réseau d'une interface réseau |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md:43]:
```suggestion
| Traffic-Generic-Id   | Net-Hirschmann-Standard-Traffic-Generic-Id-SNMP-custom   | Contrôle le trafic réseau d'une interface réseau |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md [hunk 5]
```diff
@@ -0,0 +1,569 @@
...
+| Traffic-Generic-Name | Net-Hirschmann-Standard-Traffic-Generic-Name-SNMP-custom | Contrôle le traffic réseau d'une interface réseau |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md:44]:
```suggestion
| Traffic-Generic-Name | Net-Hirschmann-Standard-Traffic-Generic-Name-SNMP-custom | Contrôle le trafic réseau d'une interface réseau |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md [hunk 6]
```diff
@@ -0,0 +1,569 @@
...
+| Traffic-Global       | Net-Hirschmann-Standard-Traffic-Global-SNMP-custom       | Contrôle le traffic réseau d'une interface réseau |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md:45]:
```suggestion
| Traffic-Global       | Net-Hirschmann-Standard-Traffic-Global-SNMP-custom       | Contrôle le trafic réseau d'une interface réseau |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md [hunk 7]
```diff
@@ -0,0 +1,569 @@
...
+> Concerne les modèles de service suivants : Traffic-Generic-Id, Traffic-Generic-Name, Traffic-Global
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+### Configuration SNMP
+
+Afin de superviser votre **Hirschmann switch** en SNMP, il est nécessaire de configurer l'agent sur l'équipement.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md:117]: prérequis standard

File: pp/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md [hunk 8]
```diff
@@ -0,0 +1,568 @@
...
+
+<Tabs groupId="sync">
+<TabItem value="Net-Hirschmann-Standard-SNMP-custom" label="Net-Hirschmann-Standard-SNMP-custom">
+
+| Service Alias | Service Template                              | Service Description                                 |
+|:--------------|:----------------------------------------------|:----------------------------------------------------|
+| Cpu           | Net-Hirschmann-Standard-Cpu-SNMP-custom       | Check the rate of utilization of CPU for the device |
+| Hardware      | Net-Hirschmann-Standard-Hardware-SNMP-custom  | Check hardware environment                          |
+| Memory        | Net-Hirschmann-Standard-Memory-SNMP-custom    | Check device memory usage                           |
+| Processes     | Net-Hirschmann-Standard-Processes-SNMP-custom | Check process number of device                      |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md:32]:
```suggestion
| Processes     | Net-Hirschmann-Standard-Processes-SNMP-custom | Check the number of processes for the device                      |
```

File: pp/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md [hunk 9]
```diff
@@ -0,0 +1,568 @@
...
+
+> The services listed above are created automatically when the **Net-Hirschmann-Standard-SNMP-custom** host template is used.
+
+</TabItem>
+<TabItem value="Not attached to a host template" label="Not attached to a host template">
+
+| Service Alias        | Service Template                                         | Service Description                   |
+|:---------------------|:---------------------------------------------------------|:--------------------------------------|
+| Configuration        | Net-Hirschmann-Standard-Configuration-SNMP-custom        | Check configuration                   |
+| Traffic-Generic-Id   | Net-Hirschmann-Standard-Traffic-Generic-Id-SNMP-custom   | Check traffic of an network interface |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md:42]:
```suggestion
| Traffic-Generic-Id   | Net-Hirschmann-Standard-Traffic-Generic-Id-SNMP-custom   | Check traffic of a network interface |
```

File: pp/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md [hunk 10]
```diff
@@ -0,0 +1,568 @@
...
+| Traffic-Generic-Name | Net-Hirschmann-Standard-Traffic-Generic-Name-SNMP-custom | Check traffic of an network interface |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md:43]:
```suggestion
| Traffic-Generic-Name | Net-Hirschmann-Standard-Traffic-Generic-Name-SNMP-custom | Check traffic of a network interface |
```

File: pp/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md [hunk 11]
```diff
@@ -0,0 +1,568 @@
...
+| Traffic-Global       | Net-Hirschmann-Standard-Traffic-Global-SNMP-custom       | Check traffic of an network interface |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md:44]:
```suggestion
| Traffic-Global       | Net-Hirschmann-Standard-Traffic-Global-SNMP-custom       | Check traffic of a network interface |
```

File: pp/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md [hunk 12]
```diff
@@ -0,0 +1,568 @@
...
+> Applies to the following service templates: Traffic-Generic-Id, Traffic-Generic-Name, Traffic-Global
+
+</TabItem>
+</Tabs>
+
+## Prerequisites
+
+### SNMP Configuration
+
+To use this pack, the SNMP service must be properly configured on your **Hirschmann switch** equipment.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md:116]: standard prerequisites

Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md:116]:
```suggestion
The SNMP agent must be configured and activated on the host. Please refer to the official documentation from the manufacturer/publisher.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md (hunk 7 above)
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-hirschmann-snmp.md:117]:
```suggestion
L'agent SNMP doit être activé et configuré sur l'équipement. Veuillez vous référer à la documentation officielle du constructeur/éditeur.
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**